int32 id
int32 new_id
int32 angle_offset
int32 angle_min
int32 angle_max
float32 voltage_min
float32 voltage_max
float32 temperature_limit
bool torque_enabled
bool led_enabled
bool led_error_temp  
bool led_error_voltage 
bool led_error_locked  
---
bool success
